QRCoder/Base64QRCode.cs

```diff
@@ -56,121 +67,80 @@ public string GetGraphic(int pixelsPerModule, string darkColorHtmlHex, string li
-    [System.Runtime.Versioning.SupportedOSPlatform("windows")]
-    public string GetGraphic(int pixelsPerModule, Color darkColor, Color lightColor, Bitmap icon, int iconSizePercent = 15, int iconBorderWidth = 6, bool drawQuietZones = true, ImageType imgType = ImageType.Png)
-    {
-        var qr = new QRCode(QrCodeData);
-        var base64 = string.Empty;
-        using (var bmp = qr.GetGraphic(pixelsPerModule, darkColor, lightColor, icon, iconSizePercent, iconBorderWidth, drawQuietZones))
-        {
-            base64 = BitmapToBase64(bmp, imgType);
-        }
-        return base64;
-    }
-
-    /// <summary>
-    /// Converts a bitmap to a base64-encoded string.
-    /// </summary>
-    /// <param name="bmp">The bitmap to convert.</param>
-    /// <param name="imgType">The type of image (PNG, JPEG, GIF).</param>
-    /// <returns>Returns the base64-encoded string representation of the bitmap.</returns>
-    [System.Runtime.Versioning.SupportedOSPlatform("windows")]
-    private static string BitmapToBase64(Bitmap bmp, ImageType imgType)
+    public string GetGraphic(int pixelsPerModule, Color darkColor, Color lightColor, bool drawQuietZones = true)
     {
-        var iFormat = imgType switch
-        {
-            ImageType.Png => ImageFormat.Png,
-            ImageType.Jpeg => ImageFormat.Jpeg,
-            ImageType.Gif => ImageFormat.Gif,
-            _ => ImageFormat.Png,
-        };
-        using var memoryStream = new MemoryStream();
-        bmp.Save(memoryStream, iFormat);
-        return Convert.ToBase64String(memoryStream.ToArray(), Base64FormattingOptions.None);
+        using var pngCoder = new PngByteQRCode(QrCodeData);
+        var pngData = pngCoder.GetGraphic(pixelsPerModule, darkColor, lightColor, drawQuietZones);
+        return Convert.ToBase64String(pngData, Base64FormattingOptions.None);
     }
 
     /// <summary>
     /// Specifies the type of image to generate.
     /// </summary>
+    [Obsolete("ImageType enum is obsolete. Only PNG format is supported.")]
     public enum ImageType
     {
         /// <summary>
         /// Graphics Interchange Format (GIF) image format, a bitmap image format with limited color support
         /// </summary>
-        [System.Runtime.Versioning.SupportedOSPlatform("windows")]
         Gif,
         /// <summary>
         /// Joint Photographic Experts Group (JPEG) image format, a lossy compressed image format
         /// </summary>
-        [System.Runtime.Versioning.SupportedOSPlatform("windows")]
         Jpeg,
         /// <summary>
         /// Portable Network Graphics (PNG) image format, a lossless raster graphics format
         /// </summary>
         Png
     }
-
 }
 
 /// <summary>
 /// Provides static methods for creating base64-encoded QR codes.
 /// </summary>
 public static class Base64QRCodeHelper
 {
+    /// <summary>
+    /// Creates a base64-encoded QR code with a single function call.
```

Comment:
```suggestion
    /// Creates a base64-encoded QR code as a one-shot operation.
```
"function call" is for imperative languages, in OOP it should be "method call".

In .NET for the hash functions, etc. they used one-shot, so we could use that here too.